src/test/java/string/StringTest.java

```diff
@@ -0,0 +1,51 @@
+package string;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+
+    @DisplayName("split 메서드를 사용해 문자열을 나눈다.")
+    @Test
+    void splitByCommas() {
+        String str = "1,2";
+        String[] results = str.split(",");
```

Comment:
```suggestion
        String[] results = "1,2".split(",");
```

굳이 로컬 변수로 구현할 필요가 있을까?